easybuild/easyconfigs/p/PRIMUS/PRIMUS-1.9.0.eb

```diff
@@ -0,0 +1,25 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author: Guilherme Balzana
+# Research Computing Services - University of Cambridge
+
+easyblock = 'Tarball'
+
+name = 'PRIMUS'
+version = '1.9.0'
+
+homepage = 'https://primus.gs.washington.edu/primusweb/res/form.html'
+description = """PRIMUS: Pedigree Reconstruction and Identification of a
+    Maximum Unrelated Set
+"""""
```

Comment:
You can one-line this (we allow for 120 columns in easyconfigs) (and i think the extra tab/space would be part of the description, which might look ugly in the actual module file).